dataprofiler/profilers/profile_builder.py

```diff
@@ -670,7 +706,67 @@ def _restore_data_labelers(self, data_labeler=None):
         :param data_labeler: unstructured data_labeler
         :type data_labeler: DataLabeler
         """
-        raise NotImplementedError()
+        # Restore data labeler for options
+        use_data_labeler = True
+        data_labeler_options = None
+        if self.options and isinstance(self.options, (StructuredOptions,
+                                                      UnstructuredOptions)):
+            data_labeler_options = self.options.data_labeler
+            use_data_labeler = data_labeler_options.is_enabled
+
+        if data_labeler_options.is_enabled \
+                and data_labeler_options.data_labeler_object is None:
```

Comment:
This would error if the above if statement wasn't entered, which yes, should always happen since we set options by default, but if thats the case, is the above if statement even necessary? Because the only way it doesn't get entered is unconventional user behavior, which at that point would save one unforseen error for another. Thoughts?

Reply:
i realized it was sort of a copy error between the struct / unstruct version, new should fix i hope.